services/src/main/java/org/fao/geonet/api/records/editing/InspireValidatorUtils.java

```diff
@@ -525,5 +503,37 @@ public static String submitFile(String serviceEndpoint, InputStream record, Stri
         }
     }
 
-}
+    private static CloseableHttpClient getHttpClient(SettingManager settingMan) {
```

Comment:
Not sure, but maybe can be used `GeonetHttpRequestFactory` methods, but if not looks fine.

Reply:
https://github.com/geonetwork/core-geonetwork/blob/0015e37bd49390d350a9b2fe552aecab7f661fc4/services/src/main/java/org/fao/geonet/api/records/editing/InspireValidatorUtils.java#L495